<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="defaultPage">
     <widget class="QPushButton" name="pushButton_Search">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string> 검 색</string>
      </property>
     </widget>
     <widget class="QTreeView" name="treeView_PhoneBook">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>341</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Add">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string> 추 가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Delete">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string> 제 거</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_LoadFile">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>310</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>불러오기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SaveFile">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>280</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>저장하기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Exit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>340</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>  종 료   </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="detailPage">
     <widget class="QPushButton" name="pushButton_Goback">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Favorite">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>즐겨찾기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Save">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>저 장</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>441</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="labelAlignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_Name">
         <property name="text">
          <string>이 름</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_Name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Call">
         <property name="text">
          <string>번 호</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_Call"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_Email">
         <property name="text">
          <string>E-mail</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_Email"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_Location">
         <property name="text">
          <string>주 소</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_Location"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_Birthday">
         <property name="text">
          <string>생 일</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDateEdit" name="dateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_SNS">
         <property name="text">
          <string>Instagram</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_SNS"/>
       </item>
       <item row="7" column="1">
        <widget class="QTextEdit" name="textEdit_Memo"/>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_SNS">
         <property name="text">
          <string>연 결</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> 메 모</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
